crates/bevy_cursor_icon/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "bevy_cursor_icon"
```

Comment:
I think the real problem here is that bevy_window isn't serving the role as a backend-agnostic layer. `bevy_asset` requires `bevy_window` because it is using `bevy_window::ANDROID_APP`. `ANDROID_APP` is not a backend-agnostic  window API.

I'm going to play around with relocating that instead, freeing up `bevy_window` to depend on `bevy_asset`.